fog/api/tests/fog_types.rs

```diff
@@ -198,38 +198,17 @@ fn tx_out_record_round_trip() {
 
     run_with_several_seeds(|mut rng| {
         let fog_txout = fog_types::view::FogTxOut::from(&TxOut::sample(&mut rng));
-
-        let commitment_bytes: &[u8; 32] = fog_txout.amount.commitment.as_ref();
-        let test_val = fog_types::view::TxOutRecord {
-            tx_out_amount_commitment_data: commitment_bytes.to_vec(),
-            tx_out_amount_masked_value: fog_txout.amount.masked_value,
-            tx_out_target_key_data: fog_txout.target_key.as_bytes().to_vec(),
-            tx_out_public_key_data: fog_txout.public_key.as_bytes().to_vec(),
-            tx_out_global_index: rng.next_u64(),
+        let meta = fog_types::view::FogTxOutMetadata {
```

Comment:
Suggest adding a comment here for what FogTxOutMetaData

Reply:
there was a comment like this in the (original) PR: https://github.com/mobilecoinfoundation/fog/pull/71#discussion_r625366668

we could add a code comment, it would say something like "Gather metadata for FogTxOut" ? does that sound good to you?